src/meson.build

```diff
@@ -198,7 +198,7 @@ libeos_shell_fx = library('eos-shell-fx',
   sources: libeos_shell_fx_sources,
   dependencies: libeos_shell_fx_deps,
   include_directories: conf_inc,
-  cpp_args: ['-std=c++11'],
+  cpp_args: ['-std=c++14'],
```

Comment:
Don't we need C++ 17?

Reply:
This is a good point. That said ... I was planning on doing a C API for libwobbly so that we can do away with a lot of this C++ business in the shell. I'll close this PR for now and open a new one on the subtask.